install/kubernetes/cilium/charts/agent/templates/daemonset.yaml

```diff
@@ -60,6 +60,9 @@ spec:
             path: /healthz
             port: {{ .Values.global.agent.healthPort }}
             scheme: HTTP
+            httpHeaders:
```

Comment:
I don't think this is strictly necessary right now, `:9876/healthz` is served by a separate minimal HTTP server which will always only return HTTP 200 or 500. But I guess it's good for forward compatibility, once we might extend this status check API.

Reply:
@tklauser although it is served by a minimal HTTP server there's a significant difference in the CPU consumption and memory usage by the server when setting `brief: true`